### DISCUSSION TITLE
How to use Boostrap-select

### DISCUSSION THREAD

**Discussion by pikooli** [vercel/next.js#17791] [Help] [answered]
Hi i try to do a pet project with react and nextjs, i use bootstrap and come across this libary.
It look cool and i would like to test it, but when i "npm install bootstrap-select" it get installed but i cannot use the className :
```
<select className="selectpicker">
  <option>Mustard</option>
  <option>Ketchup</option>
  <option>Barbecue</option>
</select>
```
this give me the default select menu.

how can i make this work ?

**Comment by jamesmosier** [comment, marked as answer]
You may want to look at using [react-bootstrap](https://react-bootstrap.netlify.app/getting-started/introduction/) which will be easier to integrate into a Next.js project (or really any React project).

1. `npm install react-bootstrap bootstrap`
2. Import the CSS file `import 'bootstrap/dist/css/bootstrap.min.css';` once somewhere (i.e. in your page or in a [`pages/_app.js` file](https://nextjs.org/docs/advanced-features/custom-app)).
3. Use the component you want:

```js
import Form from 'react-bootstrap/Form';

<Form.Control as="select" custom>
  <option>1</option>
  <option>2</option>
  <option>3</option>
  <option>4</option>
  <option>5</option>
</Form.Control>
```

**Reply by pikooli** [reply]
Thanks you for your answer :D i will check that.

i did install react-bootstrap but i was toying with other library to :D
